documents/processes/review/README.md

```diff
@@ -0,0 +1,22 @@
+# Review Process
+
+This document describes the steps you should follow when reviewing a new version of AsyncAPI.
+
+## Steps
+
+1. Please, communicate the project director that you want to start the review process.
```

Comment:
1. Replace current "Please, communicate"...to "Let the project director know that you want to start the review process."

Reply:
Done. Thanks!